scripts/devops_tasks/common_tasks.py

```diff
@@ -48,6 +48,7 @@
     "azure-synapse"
 ]
 META_PACKAGES = ["azure", "azure-mgmt", "azure-keyvault"]
+DEPRECATED_META_PACKAGES = ["azure", "azure-mgmt",]
```

Comment:
So. We aren't adding these to `OMITTED_CI_PACKAGES ` because we want the package CREATION to still work, correct?